connector_dicom/models/res_partner.py

```diff
@@ -0,0 +1,121 @@
+# Copyright (C) 2020 Konos
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import api, fields, models
+from datetime import datetime
+import requests
+import logging
+_logger = logging.getLogger(__name__)
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+
+    dicom_score = fields.Char(
+        compute="_get_dicom", inverse="_set_dicom", string="Dicom Score")
+    dicom_date = fields.Date(
+        compute="_get_dicom", inverse="_set_dicom", string="Dicom Date")
+    dicom_count = fields.Integer(
+        compute="_get_dicom_count", string="Dicom Count")
+    dicom_ids = fields.Many2many(
+        "partner.dicom", string='Dicoms', compute="_get_dicom_count",
+        readonly=True, copy=False)
+
+    def _get_dicom(self):
+        dicom = self.env['partner.dicom']
+        for record in self:
+            partner_dicom = dicom.search([('partner', '=', record.id)],
+                                         limit=1, order='date desc')
+            if partner_dicom:
```

Comment:
AFAIK you must ensure that your method always sets values on the computed fields.
I suggest just removing the `if`. In case no record is returned, it will set a None value on the fields.